packs/osx-attacks.conf

```diff
@@ -552,6 +552,22 @@
       "version" : "2.8.0",
       "description" : "Find shell processes that have open sockets and no open files or TTY (https://clo.ng/blog/osquery_reverse_shell/)",
       "value" : "Behavioral detection for potential reverse shells"
+    },
+    "OSX_ColdRoot_RAT_Launchd": {
+      "query" : "select * from launchd where name = 'com.apple.audio.driver.plist';",
+      "interval" : "3600",
+      "version" : "1.4.5",
+      "description" : "ColdRoot OSX Malware (https://objective-see.com/blog/blog_0x2A.html)",
+      "value" : "Artifacts created by this malware"
+    },
+    "OSX_ColdRoot_RAT_Files": {
+      "query" : "select * from file \
+        where path like '/private/var/tmp/com.apple.audio.driver.app/' OR \
```

Comment:
You are using like here with no wildcards so this should probably just be `=`.

You can also simplify this query with `in`:

```SQL
select * from file where path in ('/private/var/tmp/com.apple.audio.driver.app', '/private/var/tmp/com.apple.audio.driver.app/Contents/MacOS/conx.wol');
```